app/src/main/java/com/github/se/cyrcle/model/parking/Parking.kt

```diff
@@ -26,8 +28,9 @@ data class Parking(
     val rackType: ParkingRackType,
     val protection: ParkingProtection,
     val price: Double,
-    val hasSecurity: Boolean
-    // TODO: Add list of reviews when implemented
+    val hasSecurity: Boolean,
+    var nbReviews: Int = 0,
+    var avgScore: Double = 1.0
```

Comment:
Setting the default avgScore to be 1.0 is missleading. A user will think a new spot is a badly rated one.

I think Double.NaN is a better idea, and then change the cardScreen so that if the avgScore is Nan, display something to indicate than the rating is undefined rather than 0.0 or 1.0

Reply:
Good call. I'll implement this right away

Reply:
One slight issue: NaN causes problems with serialization. Does it seem like an appropriate fix to you if I put the default value at 0.0, and ensure that the text on CardScreen displays "No Reviews Yet" if nbReviews is 0? This way the default value is never shown to users

Reply:
Sounds reasonable 👍